tests/null_object_tests.py

```diff
@@ -87,3 +95,99 @@ class SimplePage(Page):
         self.driver.refresh()
         with hiro.Timeline().scale(10):
             self.assertEqual(page.id_el_changed, NonexistentElement())
+
+    # test object equivalence/nonequivalence test that expected parameters are displayed
+
+    def test_non_existence_eq_element(self):
+        class SimplePage(Page):
+            id_el = Element(Locators.ID, "simple_id")
+            other_el = Element(Locators.ID, "other_id")
+            another_el = Element(Locators.ID, "another_id")
+
+        uri = make_temp_page(ElementTest.page_content)
+        page = SimplePage(self.driver, uri)
+        non_exist_el = NonexistentElement()
+
+        first_el_text =non_exist_el.__eq__(page.id_el.text)
+        second_el_text = non_exist_el.__eq__(page.other_el.text)
+        third_el_text = non_exist_el.__eq__(page.another_el.text)
+
+        #print first_el_text == second_el_text
+        self.assertEqual(first_el_text == third_el_text, second_el_text == third_el_text, "NonExistence __eq__ unit test has failed")
+
+    def test_non_existence_nq_element(self):
+        class SimplePage(Page):
+            class_el = Element(Locators.CLASS_NAME, "simple_class")
+            class_xpath_el = Element(Locators.CLASS_NAME, "simple_xpath")
+
+        uri = make_temp_page(ElementTest.page_content)
+        page = SimplePage(self.driver, uri)
+        non_exist_el = NonexistentElement()
+
+        first_el_text = non_exist_el.__ne__(page.class_el.text)
+        second_el_text = non_exist_el.__ne__(page.class_xpath_el.text)
+
+        # print first_el_text == second_el_text
+        self.assertEqual(first_el_text != second_el_text, False,
+                         "NonExistence __ne__ unit test has failed")
+
+        # test when str() is called on the object test that expected parameters are displayed
+
+    def test_non_existence_str_element(self):
```

Comment:
To test our implementation of __str__ and __repr__ you should use global methods str() and repr() like the following:
x = NonexistentElement('val1', 'val2', 'val3')
toStr = str(x)
#do some assertions on toStr based on what you initialized object 'x' with.
#Should be a similar test for __repr__